notifikasjon/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/notifikasjon/OpprettOppgaveLoeser.kt

```diff
@@ -38,7 +38,15 @@ class OpprettOppgaveLoeser(
             behov[DataFelt.VIRKSOMHET].asText()
         )
         if (oppgaveId.isNullOrBlank()) {
-            publishFail(behov.createFail("OpprettOppgave feilet"))
+            val feil = no.nav.helsearbeidsgiver.felles.Fail(
```

Comment:
Må pushe "gammel" felles-fail for at compositeEventListener skal tolke det som en feil, håndterer ikke behov.createFail